R/snapshot.R

```diff
@@ -389,9 +389,14 @@ snapshot_accept_hint <- function(variant, file) {
     name <- file.path(variant, file)
   }
 
+  code_accept <- paste0("snapshot_accept('", name, "')")
+  code_review <- paste0("snapshot_review('", name, "')")
+  link <- function(code) {
+    cli::style_hyperlink(code, paste0("rstudio:run:testthat::", code))
```

Comment:
@gaborcsardi should we be doing something so that this only appear as a link in the ide ?